README.md

```diff
@@ -134,9 +134,8 @@ stop(); // Hammer-time!
 ```
 
 * Document all methods and properties in the headers using Xcode’s documentation style. You can select a property
-or method and use `Cmd`-`/` to see this in action. Make sure to use the available markup tags like `@param`,
-`@return`, etc. (these will be auto-generated for you by Xcode). The `///` form is preferred for single line
-comments, and `/** */` for multi-line comments. Use the same formatting as in the example block above.
+or method and use `Option`+`Cmd`+`/` to see this in action. Make sure to use the available markup tags like `@param`,
```

Comment:
I think it would be good to mention the format we want. For those that aren’t using this specific version of Xcode, or Xcode at all. I.e. adding a mention like “Use `///` (three forward slashes) for single and multi-line comment.

Reply:
I actually thought it would be easier to just follow the current Xcode format, whatever that is. I agree though, that it could be unclear for someone who's using IDE. Do we know how many people are?

Reply:
AFAIK, only one person is not using Xcode at this time, so we can assume that everyone is :P

Reply:
I would also recommend `///` specifically instead of the hotkey as I expect these to not be changed soon
BTW, the markups aren't `@param` and such anymore. It's `Parameter:`, `Return:` and so on:

```
    /// Description
    ///
    /// - Parameters:
    ///   - withOperations: The array of content operations to use to display content.
    ///   - type: The desired type of the resulting `SPTHubViewController`.
    ///
    /// - Returns: The resulting `SPTHubViewController` that's initialized as `type`.
```
I would also maybe recommend a styling approach to it (like a blank line between the sections like above) as I think the generated Xcode one doesn't do so and it looks a bit weird IMO

Reply:
@rockbruno it is still `@param` for ObjC.